### PR TITLE
Reload standard libraries

### PR DESCRIPTION
**Pull request by mjyip-lucasfilm**
Added hotkey to reload standard libraries. Also added a label that displays the current shading model (if it exists).

### CODE REVIEW

File: source/MaterialXView/Viewer.cpp [hunk 1]
```diff
@@ -810,6 +806,17 @@ void Viewer::loadDocument(const mx::FilePath& filename, mx::DocumentPtr librarie
         for (const auto& renderablePath : renderablePaths)
         {
             mx::ElementPtr elem = doc->getDescendant(renderablePath);
+            if (!elem->getAttribute("node").empty())
```
Comment (jstone-lucasfilm) [inline source/MaterialXView/Viewer.cpp]: There are other situations in which we'll need to update the "shading model" display for the current material, e.g. when the user changes the selected material or selected geometry.  To keep everything in sync, I would recommend extending the existing Viewer::updatePropertyEditor method, making it general enough to include your new functionality as well.  One potential name for the new, more comprehensive method would be Viewer::updateDisplayedProperties.

Comment (mjyip-lucasfilm) [reply source/MaterialXView/Viewer.cpp]: Sounds good! At the moment, each time a new material is chosen, loadDocument is called and then loadDocument calls updateDisplayedProperties. Do you suggest I move the updateDisplayedProperties outside of loadDocument?

Comment (jstone-lucasfilm) [reply source/MaterialXView/Viewer.cpp]: My thinking is that we would replace updatePropertyEditor with the more general updateDisplayedProperties, and then call the new method from all of the old locations where we used to call updatePropertyEditor.  That would include loadDocument, but also other locations where the user is switching between materials or geometries.

File: source/MaterialXView/Viewer.cpp [hunk 2]
```diff
@@ -1568,7 +1588,21 @@ void Viewer::computeCameraMatrices(mx::Matrix44& world,
     world *= mx::Matrix44::createTranslation(_modelTranslation).getTranspose();
 }
 
-void Viewer::updatePropertyEditor()
+void Viewer::updateDisplayedProperties()
 {
     _propertyEditor.updateContents(this);
+    mx::TypedElementPtr elem = getSelectedMaterial() ? getSelectedMaterial()->getElement() : nullptr;
+    const std::string& node = (elem && !elem->getAttribute("node").empty()) ? elem->getAttribute("node"): "";
```
Comment (jstone-lucasfilm) [inline source/MaterialXView/Viewer.cpp]: One minor note here:  If the return value of `elem->getAttribute("node")` is empty, then we can use that empty string as the value of this variable, saving us a bit of extra logic to explicitly check for the empty string and substitute `""`.

**Pull request merged**